packages/connect-popup/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        "outDir": "lib",
```

Comment:
We should keep convention that `tsconfig.json` is used just for type-check and not for actual build. So there should be `libDev`. If you want to also build this package using `tsc` please another `tsconfig.lib.json` that will be used for builds.

Reply:
sure, fixed.